### ISSUE TITLE
Support Electron 32

### ISSUE THREAD

**Issue by johnlindquist** [microsoft/node-native-keymap#60]
I'm getting the same error for node-native-keymap as the sqlite package was getting when attempting upgrade to Electron 32:

https://github.com/WiseLibs/better-sqlite3/pull/1226

I tried forking and solving it myself, but it's just too far out of my wheelhouse to fully understand the problem. Please help :)

Maybe it's related to this?

https://github.com/electron/rebuild/issues/1044

**Comment by asamofal** [comment]
Double check that you use [v11.2.1](https://github.com/WiseLibs/better-sqlite3/releases/tag/v11.2.1) that includes [#1226](https://github.com/WiseLibs/better-sqlite3/pull/1226).

v11.2.1 works fine for me now.

**Comment by johnlindquist** [comment]
@asamofal yeah, I was implying it's been fixed in SQLite and that this repo might benefit from a similar fix

**Comment by asamofal** [comment]
@johnlindquist oh, sorry,I just got the repo confused 🤦‍♂️
